fe/src/main/java/org/apache/doris/load/DeleteHandler.java

```diff
@@ -569,6 +587,19 @@ private void checkDeleteV2(OlapTable table, Partition partition, List<Predicate>
                     sb.append(" IS NULL");
                 }
                 deleteConditions.add(sb.toString());
+            } else if (condition instanceof InPredicate) {
+                InPredicate inPredicate = (InPredicate) condition;
+                SlotRef slotRef = (SlotRef) inPredicate.getChild(0);
+                String columnName = slotRef.getColumnName();
+                StringBuilder strBuilder = new StringBuilder();
+                String notStr = inPredicate.isNotIn() ? "NOT " : "";
+                strBuilder.append(columnName).append(" " + notStr + "IN (");
+                for (int i = 1; i <= inPredicate.getInElementNum(); ++i) {
+                    strBuilder.append(inPredicate.getChild(i).toSql());
+                    strBuilder.append((i + 1 != inPredicate.getInElementNum()) ? ", " : "");
```

Comment:
the last loop is “ i == inPredicate.getInElementNum()”, should " i + 1 != inPredicate.getInElementNum()" modify "i != inPredicate.getInElementNum()"?

Reply:
oh， mybad，thanks，I will fix it